Training3/src/src/components/Task.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import { mapKeyboard } from "../constants/mapKeyboard";
+import Modal from "./Modal";
+
+function Task() {
+  const [isShowing, setIsShowing] = useState(false);
+  const [logger, setLogger] = useState([]);
+  const handleClick = () => {
+    setIsShowing((isShowing) => !isShowing);
+  };
+  const handleKeyDown = (e) => {
```

Comment:
We should move handleKeyDown into the useEffect, we only want to a listener function created or listener created whenever modal change hide -> showing